cpp/example_code/rds/README.md

```diff
@@ -0,0 +1,114 @@
+<!--Generated by WRITEME on 2023-02-15 19:59:47.927677 (UTC)-->
+# Amazon RDS code examples for the SDK for C++
+
+## Overview
+
+Shows how to use the AWS SDK for C++ to work with Amazon Relational Database Service.
+
+<!--custom.overview.start-->
+<!--custom.overview.end-->
+
+*Amazon RDS is a web service that makes it easier to set up, operate, and scale a relational database in the cloud.*
+
+## ⚠ Important
+
+* Running this code might result in charges to your AWS account.
+* Running the tests might result in charges to your AWS account.
+* We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+<!--custom.important.start-->
+<!--custom.important.end-->
+
+## Code examples
+### Single actions
+
+Code excerpts that show you how to call individual service functions.
+
+* [Create a DB instance](getting_started_with_db_instances.cpp#L483) (`CreateDBInstance`)
+* [Create a DB parameter group](getting_started_with_db_instances.cpp#L315) (`CreateDBParameterGroup`)
+* [Create a snapshot of a DB instance](getting_started_with_db_instances.cpp#L561) (`CreateDBSnapshot`)
+* [Delete a DB instance](getting_started_with_db_instances.cpp#L852) (`DeleteDBInstance`)
+* [Delete a DB parameter group](getting_started_with_db_instances.cpp#L906) (`DeleteDBParameterGroup`)
+* [Describe DB instances](getting_started_with_db_instances.cpp#L747) (`DescribeDBInstances`)
+* [Describe DB parameter groups](getting_started_with_db_instances.cpp#L277) (`DescribeDBParameterGroups`)
+* [Describe database engine versions](getting_started_with_db_instances.cpp#L713) (`DescribeDBEngineVersions`)
+* [Describe options for DB instances](getting_started_with_db_instances.cpp#L786) (`DescribeOrderableDBInstanceOptions`)
+* [Describe parameters in a DB parameter group](getting_started_with_db_instances.cpp#L656) (`DescribeDBParameters`)
+* [Describe snapshots of DB instances](getting_started_with_db_instances.cpp#L599) (`DescribeDBSnapshots`)
+* [Update parameters in a DB parameter group](getting_started_with_db_instances.cpp#L384) (`ModifyDBParameterGroup`)
+
+### Scenarios
+
+Code examples that show you how to accomplish a specific task by calling multiple
+functions within the same service.
+
+* [Get started with DB instances](getting_started_with_db_instances.cpp) 
+
+## Run the examples
+
+### Prerequisites
+
+
+
```

Comment:
Unable to test if this renders with extra line breaks or not, just calling it out.